cypress/integration/1_detectors.spec.js

```diff
@@ -119,7 +139,6 @@ describe('Detectors', () => {
     cy.contains('Detector details');
     cy.contains('Created at');
     cy.contains('Last updated time');
-    cy.contains('Successfully created detector, "test detector"');
```

Comment:
Agreed, this diff doesn't illustrate the sequence of changes (time wise) quite right. I removed it since only on success are we brought to the details page, where the existence of the new detector can be confirmed with non-ephemeral elements.